libs/src/services/contracts/ContractClient.js

```diff
@@ -59,6 +59,10 @@ class ContractClient {
     try {
       if (!this._contractAddress) {
         this._contractAddress = await this.getRegistryAddress(this.contractRegistryKey)
+        if (this._contractAddress === "0x0000000000000000000000000000000000000000") {
```

Comment:
Without this check, we would consider a contract to be 'initialized' at the zero address